app/sentinel/internal/oidc/server/server.go

```diff
@@ -35,6 +35,11 @@ import (
 //     requests to the "/secrets" endpoint.
 func Serve() {
 	mux := http.NewServeMux()
+
+	safeOps := new(SafeClient)
+	logger := new(RpcLogger)
+
+	engine := engine.New(safeOps, logger)
```

Comment:
There’s nothing inherently wrong with using `new` in Go, but it’s less commonly used in idiomatic Go code because of how it behaves compared to other memory allocation patterns in Go. The new function allocates memory but doesn't initialize the object beyond setting it to its zero value. 

This is in contrast to using a constructor-like approach (which can include more complex initialization logic).

Consider using `safeOps := &SafeClient{}`

Or if you feel Java…ey, using a custom constructor function:

```
func NewSafeClient() *SafeClient {
    return &SafeClient{
        // Initialize necessary fields here
    }
}

safeOps := NewSafeClient()
```

Also, if you don't need a pointer to the struct, you can simply assign it directly:

```
safeOps := SafeClient{}
logger := RpcLogger{}
```

Reply:
Thanks for the review, let me make it more goish not javaish :)